package.json

```diff
@@ -21,7 +21,7 @@
         "@commitlint/cli": "^17.1.2",
         "@commitlint/config-conventional": "^17.1.0",
         "@commitlint/config-nx-scopes": "^17.0.0",
-        "@deriv/eslint-config-deriv": "^1.0.0-beta.3",
+        "@deriv/eslint-config-deriv": "^1.0.0-beta.5",
```

Comment:
The sorting rule is added into this library, check this repo:
https://github.com/binary-com/eslint-config-deriv